File: CS/ProcessDuplicatesTarget/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ProcessDuplicatesTarget
{
    public partial class Form1 : DevExpress.XtraEditors.XtraForm
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            ProductReport report = new ProductReport();
            report.NoMerge();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            ProductReport report = new ProductReport();
            report.MergeByTag();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            ProductReport report = new ProductReport();
            report.MergeByValue();
        }
    }
}

File: VB/ProcessDuplicatesTarget/Form1.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms

Namespace ProcessDuplicatesTarget

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim report As ProductsReport = New ProductsReport()
            report.NoMerge()
        End Sub

        Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim report As ProductsReport = New ProductsReport()
            report.MergeByTag()
        End Sub

        Private Sub button3_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim report As ProductsReport = New ProductsReport()
            report.MergeByValue()
        End Sub
    End Class
End Namespace
